bundler/lib/dependabot/bundler/update_checker/shared_bundler_helpers.rb

```diff
@@ -65,6 +77,17 @@ def in_a_temporary_bundler_context(error_handling: true)
           error_handling ? handle_bundler_errors(e) : raise
         end
 
+        # TODO: Use/remove argument
+        def in_a_native_bundler_context(_error_handling: true)
```

Comment:
My thinking on this is that `handle_bundler_errors` _should_ be used as part of this method but it will need the interface changes to accept a string, along with passing back the error class from the native helper. I've punted on this for now as resolving it isn't specific to this component.

Reply:
Yeah nice, lets get this in and pair on refactoring the error handling Monday?